src/components/Tile/Tile.js

```diff
@@ -453,6 +454,10 @@ export class ExpandableTile extends Component {
     const content = this.getChildren().map((child, index) => {
       return React.cloneElement(child, { ref: index });
     });
+
+    // a unique ID generated for use in aria-labelledby
+    this.uid = uid();
```

Comment:
The `uid` is created in creation time instead of rendering time - Otherwise every `render()` changes the unique ID and thus redundant rendering will happen.